initramfs/initramfs.list

```diff
@@ -0,0 +1,26 @@
+# directory structure
+dir	/bin					0755 0 0
+dir	/dev					0755 0 0
+dir	/etc					0755 0 0
+dir	/lib					0755 0 0
+dir	/mnt					0755 0 0
+dir	/mnt/root				0755 0 0
+dir	/proc					0755 0 0
+dir	/root					0700 0 0
+dir	/run					0755 0 0
+dir	/sbin					0755 0 0
+dir	/sys					0755 0 0
+dir	/tmp					1777 0 0
+dir	/usr					0755 0 0
+dir	/usr/bin				0755 0 0
+dir	/usr/sbin				0755 0 0
+
+# device nodes
+nod	/dev/console				600 0 0 c 5 1
+
+# busybox
+file	/bin/busybox	initramfs/busybox	0755 0 0
+slink	/sh		/bin/busybox		0755 0 0
+slink	/bin/sh		/bin/busybox		0755 0 0
+
+file	/init		initramfs/init.sh	0755 0 0
```

Comment:
not /sbin/init ?

Reply:
Because that doesn't work :)
https://github.com/Ultimaker/linux/blob/opinicus-4.14-0/Documentation/admin-guide/kernel-parameters.txt#L3630
But more importantly
https://github.com/Ultimaker/linux/blob/opinicus-4.14-0/init/main.c#L1080